### PR TITLE
Update DG: introduction + model class diagram

### PR DESCRIPTION
**Pull request by duongphammmm**

### CODE REVIEW

File: docs/diagrams/ModelClassDiagram.puml [hunk 1]
```diff
@@ -45,43 +54,53 @@ Class Tag
...
 
-ResumeBook *--> "4" UniqueItemList
+ResumeBook *--> "5" UniqueItemList
 UniqueItemList o--> "*" Item
 
 Item *---> "1" Name
 Item *---> "*" Time
 Item *---> "1" Type
-Item *---> "1" Description
+Item *---> "*" Description
```
Comment (chrisjwelly) [inline docs/diagrams/ModelClassDiagram.puml]: Remind me: an item has many descriptions?

**Review by chrisjwelly — COMMENTED** [review]
I know the original diagrams from AB3 had arrowheads for Composition and Aggregation, but I was looking at the [website](https://nus-cs2103-ay1920s2.github.io/website/book/modeling/modelingStructures/classDiagramsIntermediate/) last night and it didn't have any:
image

I'm not fully sure what to do here.

Also another remark about our `Item` class: does our `Item` actually have `Time` and `Description`? Because technically `Resume` and `Skill` which extends `Item` does not have `Time` and `Description`.

File: docs/diagrams/ModelClassDiagram.puml (hunk 1 above)
Comment (duongphammmm) [reply docs/diagrams/ModelClassDiagram.puml]: ooooh right what i meant is either 0 or 1, will fix that 👍

Comment (helloImHai) [reply docs/diagrams/ModelClassDiagram.puml]: key constraint

**Pull request merged**